tests/utilities.py

```diff
@@ -0,0 +1,24 @@
+"""
+Assorted utilities useful for the tests.
+"""
+
+import os
+import contextlib
+
+try:
+    import urllib.request as urllib_request  # for python 3
+except ImportError:
+    import urllib2 as urllib_request  # for python 2
+
+import pytest
+import glob
+
+from .get_remote_data import get_datafile
+
+
+def get_test_file_dir():
```

Comment:
we can just put this in the test file directly -- or *maybe* make it a fixture.

And pathlib is great:

from pathlib import Path

`test_file_dir = Path(__file__).parent /  `'test_data')`